samples/client/echo_api/typescript-axios/build/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-7.15.0-SNAPSHOT
+7.13.0-SNAPSHOT
```

Comment:
please merge the latest master to re-generate all samples. that should revert this line